workflow/kfdrc-jointgenotyping-refinement-workflow.cwl

```diff
@@ -0,0 +1,317 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc-jointgenotyping-refinement-workflow
+label: Kids First DRC Joint Genotyping Workflow
+doc: |
+  # Kids First DRC Joint Genotyping Workflow
+  Kids First Data Resource Center Joint Genotyping Workflow (cram-to-deNovoGVCF). Cohort sample variant calling and genotype refinement.
+
+  Using existing gVCFs, likely from GATK Haplotype Caller, we follow this workflow: [Germline short variant discovery (SNPs + Indels)](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145), to create family joint calling and joint trios (typically mother-father-child) variant calls. Peddy is run to raise any potential issues in family relation definitions and sex assignment.
```

Comment:
Oh, one last request - perhaps sneak in the anticipated git release version, and a link to the repo. Similarly, in the readme, point to the cavatica public app. You can see how I did that in the alignment workflow

Reply:
Ok I put in the bit linking to the prospective public app in the README and for the public app I added a new tag for sbg:links to link to the prospective release. Prospective.